src/mapping/PetRadialBasisFctMapping.hpp

```diff
@@ -753,11 +778,9 @@ void PetRadialBasisFctMapping<RADIAL_BASIS_FUNCTION_T>::mapConservative(DataID i
 
     // Gets the petsc::vector for the given combination of outputData, inputData and dimension
     // If none created yet, create one, based on _matrixC
-    petsc::Vector &out = std::get<0>(
-                             previousSolution.emplace(std::piecewise_construct,
-                                                      std::forward_as_tuple(inputDataID + outputDataID * 10 + dim * 100),
-                                                      std::forward_as_tuple(petsc::Vector::allocate(_matrixC, "out"))))
-                             ->second;
+    //petsc::Vector out = petsc::Vector::allocate(_matrixC, "out").copyFrom(this->lastSolution());
```

Comment:
Dead code or important to remain here?

Reply:
Removed the dead code. Thanks for pointing it out